_docs/setup/kubernetes/quick-start.md

```diff
@@ -123,11 +123,7 @@ kubectl apply -f install/kubernetes/istio-auth.yaml
   Both options create the `istio-system` namespace along with the required RBAC permissions,
   and deploy Istio-Pilot, Istio-Mixer, Istio-Ingress, and Istio-CA (Certificate Authority).
 
-1. *Optional:* If your cluster has Kubernetes alpha features enabled, and you wish to enable a
-   [automatic injection of sidecar]({{home}}/docs/setup/kubernetes/sidecar-injection.html#automatic-sidecar-injection), install the Istio-Initializer:
-```bash
-kubectl apply -f install/kubernetes/istio-initializer.yaml
-```
+1. *Optional:* If your cluster has Kubernetes version 1.9 or greater, and you wish to enable automatic proxy injection, install the sidecar injector webhook using the instructions at ({{home}}/docs/setup/kubernetes/sidecar-injection.html#automatic-sidecar-injection).
```

Comment:
I could be mistaken, but I suspect you need alpha gates enabled?

Reply:
Mutating webhooks are beta in 1.9 and should be enabled by default. It is on GKE. Any cluster specific instructions should be documented in https://istio.io/docs/setup/kubernetes/sidecar-injection.html#prerequisites

Reply:
+ @cmluciano @jsenon who are documenting setup instructions for minkube and aws/kops and correct me

Reply:
It depends on the if the author explicitly removed it from the feature gate code. I will check the k8s code.

Reply:
Trying this locally with minikube. There might be additional parameters required despite lack of an explicit feature-gate flag.